voting/test/it/voting.it.spec.js

```diff
@@ -115,6 +409,46 @@ describe("voting contracts on orbs and ethereum", async () => {
         const winners = await electionContracts.getElectionWinners();
         expect(winners).to.have.members([v1, v2, v4, v5].map(v => v.orbsAccount.address)); // TODO - which should be voted in???
 
-        // XXXX end of flow. gamma does not enforce the results of elections on validator committee. it requies an "unsafe_" operation. consider supporting
-    })
+        // check reward  added to v1 and v2 with time based values.
+        expect(await electionContracts.getOrbsValidatorReward(electionContracts.addressWithoutChecksum(v1.address))).to.be.equal(8533); // +8533
+        expect(await electionContracts.getOrbsValidatorReward(electionContracts.addressWithoutChecksum(v2.address))).to.be.equal(8536); // +8536
+
+        console.log("Done First Election\n");
+
+        console.log("Start A Time Election as second");
```

Comment:
where is the call that switches on the 'time based' toggle?

Reply:
inside the setElectionTimeToCurrentEthereumTime. the switchTo can be called multiple times.